modules/apps/frontend-js/frontend-js-test/src/testFunctional/paths/A11yTool.path

```diff
@@ -120,6 +120,21 @@
 	<td>//*[contains(@class,'a11y-panel')]//*[contains(@class,'panel-body')]//ul//*[contains(@class,'list-group-item') and contains(.,'${key_text}')]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>VIOLATION_PANEL_TARGET_CODE_HEADER</td>
+	<td>//*[contains(@class,'a11y-panel')]//*[contains(@class,'my-3') and contains(.,'${key_text}')]</td>
```

Comment:
`my-3` is an interesting target class for code header...that's the best available?

Reply:
yeah :\ 
![image](https://user-images.githubusercontent.com/28836926/142512666-1a8561c6-9d47-4a1f-a564-9f0fe8d8c68b.png)

Reply:
ic...looks like `my-3` is bootstrap code for margin top and bottom, 3 being 1 rem. 
![image](https://user-images.githubusercontent.com/35235266/142539715-bf38af2a-8acc-44e9-897d-1ec3e31d8bfe.png)

https://stackoverflow.com/questions/50750931/what-do-css-classes-like-my-2-my-lg-0-mr-sm-2-do-in-bootstrap-4

I'd like to avoid using bootstrap or generic css classes not really tied to describing the element we're locating. 

what about if we had something similar to:
```
<td>VIOLATION_PANEL_OCCURRENCE_TARGET_SELECTOR</td>
	<td>//*[contains(@class,'a11y-panel')]//*[contains(@class,'code-block') and contains(@aria-label,'Target Selector')]//*[contains(., '${key_text}')]</td>
	<td></td>
```
But it would reference the sibling right above it? using `preceding-sibling::` I think that would describe the element you're targeting better.

Same thing with:
```
	<td>VIOLATION_PANEL_OCCURRENCE_CODE_ELEMENT</td>
	<td>//*[contains(@class,'a11y-panel')]//*[contains(@class,'code-block') and contains(@aria-label,'Code of the Element')]//*[contains(., '${key_text}')]</td>
	<td></td>
```

Reply:
i see i see... how does this look?

`//*[contains(@class,'a11y-panel')]//*[contains(@class,'code-block') and contains(@aria-label,'${key_text}')]//preceding-sibling::div[contains(.,'${key_header}')]`

Reply:
This looks better, and maybe use `key_label` instead of `text`
```
//*[contains(@class,'a11y-panel')]//*[contains(@class,'code-block') and contains(@aria-label,'${key_label}')]//preceding-sibling::div[contains(.,'${key_header}')]
```